exercises/concept/black-jack/.meta/config.json

```diff
@@ -1,6 +1,6 @@
 {
   "blurb": "Learn about comparisons by implementing some Black Jack judging rules.",
-  "authors": ["Ticktakto", "Yabby1997", "limm-jk", "OMEGA-Y", "wnstj2007"],
+  "authors": ["Ticktakto", "Yabby1997", "limm-jk", "OMEGA-Y", "wnstj2007", "pranasas"],
```

Comment:
Did I get it right that this ought to be a handle used on exercism.org?

Reply:
@pranasziaukas  -- It should be your GitHub handle.  And if you are not using GitHub to log into exercism, then we'll need to make sure that we track your PRs for credit to your exercism account.  LMK if you are not getting credited, and I will make sure that you (eventually) do.  An author credit is 20 IIRC.  The code to reconcile the exercism account different from  GitHub account is still in process.

Reply:
```suggestion
  "authors": ["Ticktakto", "Yabby1997", "limm-jk", "OMEGA-Y", "wnstj2007", "pranasziaukas"],
```

Reply:
Got it @BethanyG. My Exercism handle is [`pranasas`](https://exercism.org/profiles/pranasas) and it is separate from GitHub (I have my GitHub handle listed under Exercism settings though).
No reputation from contributions so far. It's fine if it will bet credited *eventually*.